README.md

```diff
@@ -22,6 +22,12 @@ npm install
 
 # run the build script to build required assets
 npm run build
+
```

Comment:
I would like to move this to a dedicated PR as well and implement it as a reusable NPM script. It also needs to be documented in the docs as well.